pkg/storaged/content-views.jsx

```diff
@@ -422,7 +423,7 @@ function append_partitions(client, rows, level, block) {
         ];
 
         rows.push(
-            <CockpitListing.ListingRow columns={cols} />
+            <CockpitListing.ListingRow columns={cols} key="free-space" />
```

Comment:
There can be more than one free space entry and we should give them separate keys.

Reply:
Fixed in "storage: More React keys"